src/style.css

```diff
@@ -0,0 +1,65 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: block;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star:first-child {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--5 .stars__star:nth-child(-n+5) {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
@ivaZaiets You have too many line duplicates what makes your code to be read harder. See here how you can rewrite to have the same affect but with less lines.
![image](https://github.com/mate-academy/layout_stars/assets/45945732/6b65c6e5-f928-44ad-9be3-d081be0f83cf)